cmake/Modules/MacroQbtCompilerSettings.cmake

```diff
@@ -24,8 +24,8 @@ macro(qbt_set_compiler_options)
             #"-Wno-error=sign-conversion -Wno-error=float-equal"
         )
 
+        # GCC 4.8 has problems with std::array and its initialization
```

Comment:
Is this still relevant, since you're also dropping the glibc 2.20 check? At least looking at Ubuntu releases, there was no  release where GCC <= 4.8 and also glibc >= 2.20. So if the assumption is that qBittorrent is being built on a software stack recent enough that there is no need to check for glibc < 2.20, then there is no need to check for GCC <= 4.8 either.

I don't think it is that much of a problem to just bump the minimum required GCC  version to 4.9 or something.

Reply:
I think you're misunderstanding the reason I removed the check. glibc won't care if we pass DEFAULT_SOURCE. It's not a version requirement bump, rather just the removal of an unnecessary check.

Reply:
Thanks for clarifying.